ethereum/contracts/script/utils/NetworkConfig.s.sol

```diff
@@ -63,7 +63,8 @@ contract NetworkConfig is Script {
     EnvironmentType public currentEnvironmentType;
     NetworkDetail public currentNetworkDetail;
 
-    string public pathToDeploymentFile = string(abi.encodePacked(vm.projectRoot(), "/contracts-addresses.json"));
+    string public pathToDeploymentFile =
+        string(abi.encodePacked(vm.projectRoot(), "/../bindings/contracts-addresses.json"));
```

Comment:
Why would we need the physical file still? If it is needed like this, it should be in repo root, and not inside the bindings.

Reply:
This is to expose the contents natively in Solidity. Similar to what the crate for Rust does.